external-crates/move/crates/move-compiler/tests/linter/incorrect_constant_naming.move

```diff
@@ -0,0 +1,16 @@
+module 0x42::M {
+    // Incorrectly named constants
+    const Another_BadName: u64 = 42; // Should trigger a warning
+    const YetAnotherName: u64 = 777; // Should not trigger a warning
```

Comment:
Similarly, remove any valid names from this test

Reply:
refactored